src/lib/containers/EntityLink.tsx

```diff
@@ -75,6 +76,9 @@ export const EntityLink = (props: EntityLinkProps) => {
       )
     }
   } else if (error) {
+    if (entityId) {
+      return <ErrorChip chipText={entityId} error={error} />
+    }
```

Comment:
Here's the first place we'll use this component. Show the new error chip if we encounter an error and have an ID that we can show in the chip.